tests/test_validate_schedule.py

```diff
@@ -37,13 +37,17 @@ def correct_schedule():
 def test_generate_validation_report_for_correct_schedule(correct_schedule):
     correct_report = {
         'schedule_level': {'is_valid_schedule': True, 'invalid_stages': [], 'has_valid_services': True,
-                           'invalid_services': []},
+                           'invalid_services': [], 'headways': {'has_zero_min_headways': False}},
         'service_level': {
             'service': {'is_valid_service': True, 'invalid_stages': [],
                         'has_valid_routes': True, 'invalid_routes': []}},
         'route_level': {
-            'service': {'1': {'is_valid_route': True, 'invalid_stages': []},
-                        '2': {'is_valid_route': True, 'invalid_stages': []}}},
+            'service': {'1': {'is_valid_route': True, 'invalid_stages': [],
```

Comment:
A quick note on an edge case that we ran into in the NZ ABM was with (inter-regional) buses which often only have one service / trip a day. We ended up accounting for them by hardcoding all services with a service/trip length of 1 to a 24-hour headway, not sure how often this would occur however something that we ran into.

Reply:
Yeah that's a very good point, in the old version those would default to a 0 headway. I settled on leaving the NaT/NaN because I didn't want to presume.. My thinking is, we could leave the empty for the user to interpret - I think it's not impossible there to be a service that shows up in the schedule but runs once a week and then the headway should ideally be 7*24*60 right? But we can't know about it because we lost the part of GTFS that could tell us about it. All throughout the genet schedule though, we talk about gtfs DAY and the services are assumed to be running in a day, so a 24h default would make sense.

What do you think would make most sense / be the most convenient for a potential user?